File: packages/engine/src/cpus/6502/instructions/clv.js
import { CycleType } from '..'

/**
 * Clear oVerflow
 *
 * @example
 * MODE          SYNTAX        HEX LEN TIM
 * Implied       CLV           $B8  1   2
 */
export function* clv() {
  this.overflow = false
  yield CycleType.GET
}

File: packages/engine/src/cpus/6502/instructions/clv.zig
const std = @import("std");
const CPU = @import("../cpu.zig").CPU;
const Addressing = @import("../opcode.zig").Addressing;

// Clear oVerflow
//
// MODE          SYNTAX        HEX LEN TIM
// Implied       CLV           $B8  1   2

pub fn clv(self: *CPU, addressing: Addressing) void {
    _ = addressing;
    self.v = false;
}
